packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/import_saved_objects.test.ts

```diff
@@ -72,7 +72,17 @@ describe('#importSavedObjectsFromStream', () => {
   let savedObjectsClient: jest.Mocked<SavedObjectsClientContract>;
   let typeRegistry: jest.Mocked<ISavedObjectTypeRegistry>;
   const namespace = 'some-namespace';
-
+  const mockLog = {
```

Comment:
There is an existing mock for logging, prefer using that to a custom mock. 
```
import { loggerMock, type MockedLogger } from '@kbn/logging-mocks';
```

TAL at how it's done in https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/export/saved_objects_exporter.test.ts